### ISSUE TITLE
Parse error

### ISSUE THREAD

**Issue by leximayfield** [gf3/IRC-js#16]
Ran into this parse error.  Any thoughts?  Using node.js 0.3.4 and irc-js 0.2.11 from npm.

```
[Mon, 10 Jan 2011 16:23:20 GMT] ERROR      [ERROR] Failed parsing ':*.quakenet.org MODE #altdeath +v Typone'

/home/alexmax/local/lib/node/.npm/irc-js/0.2.11/package/lib/irc.js:136
          throw err
          ^
Error: treeWalker: parse failed: 0 parse already failed
    at err (/home/alexmax/local/lib/node/.npm/irc-js/0.2.11/package/node_modules/PanPG/PanPG_util.js:157:9)
    at Object.treeWalker (/home/alexmax/local/lib/node/.npm/irc-js/0.2.11/package/node_modules/PanPG/PanPG_util.js:104:10)
    at Object.walk (/home/alexmax/local/lib/node/.npm/irc-js/0.2.11/package/lib/walker.js:36:21)
    at Object.compile (/home/alexmax/local/lib/node/.npm/irc-js/0.2.11/package/lib/compiler.js:8:17)
    at IRC.parseMessage (/home/alexmax/local/lib/node/.npm/irc-js/0.2.11/package/lib/irc.js:116:28)
    at Socket.<anonymous> (native)
    at Socket.emit (events.js:31:17)
    at Socket._onReadable (net.js:610:31)
    at IOWatcher.onReadable [as callback] (net.js:155:10)
```

**Comment by gf3** [comment]
I bet it's the asterisk in the domain.

**Comment by gf3** [comment]
![BOOM](http://f.cl.ly/items/1o2B1s2D273c2i2L2y1T/Screen%20shot%202011-01-10%20at%206.38.53%20PM.png)

Fixed in: eda73fe530220b4d5b0c15cc2a209416be32fd10
